packages/app/src/navigation/SidebarNavigationHeader.vue

```diff
@@ -6,6 +6,8 @@
     :distance="8"
   >
     <div
+      role="button"
+      :aria-label="t('testingType.modalTitle')"
```

Comment:
Setting an explicit label here will cause a screen reader to ignore the button's text content (project/branch). That'd probably be fine if the project were announced elsewhere (part of the title?), but I don't think it is. 

We could compose the label (`${project}, ${branch}, ${t('testingType.modalTitle')}`)? This button feels overloaded.

Reply:
Agh I agree, this button is definitely overloaded 

I think we could get away with `${project} - ${t('testingType.modalTitle')` with the rationale being: The user isn't depending on Cypress to tell them which git branch they're on, but they definitely are depending on Cypress to indicate the project name and how to switch testing types

We _might_ be able to get away with just `:aria-label="t('testingType.modalTitle')"` given that this PR adds a `title` attribute to the iframe with the projectId:

<img width="509" alt="Screen Shot 2022-05-16 at 12 14 14 PM" src="https://user-images.githubusercontent.com/26726429/168625857-b3cf8600-c1ee-4352-8574-912afc42b204.png">

@tbiethman what do you think? I can go either way